edb/ir/scopetree.py

```diff
@@ -209,15 +209,15 @@ def descendant_namespaces(self) -> typing.Set[str]:
         return namespaces
 
     @property
-    def unfenced_descendants(self) -> typing.Iterator['ScopeTreeNode']:
+    def unfenced_descendants(self) -> Iterator['ScopeTreeNode']:
         """An iterator of node's unfenced descendants including self."""
         yield self
         for child in tuple(self.children):
             if not child.fenced:
                 yield from child.unfenced_descendants
 
     @property
-    def strict_unfenced_descendants(self) -> typing.Iterator['ScopeTreeNode']:
+    def strict_unfenced_descendants(self) -> Iterator['ScopeTreeNode']:
```

Comment:
We can probably also fix things like `Iterator['ScopeTreeNode']` to `Iterator[ScopeTreeNode]` if that's easy to handle in the script you have; and if not this can wait/be done manually.

Reply:
OK, replaced all string annotations with proper names.

Reply:
Awesome!